### PR TITLE
some tweaks to jaspReport

### PR DESCRIPTION
**Pull request by JorisGoosen**
make top notification better in jaspReport
make sure Desktopmain checks for --webEngineArgs as that is nowadays neccesary to debug via chrome

### CODE REVIEW

**Comment by JorisGoosen** [comment]
This can be merged after 0.17.2 is released as well, no hurry.

**Comment by JorisGoosen** [comment]
https://github.com/jasp-stats/jaspBase/pull/126 belongs with this
(Its commited as submodule in jaspBase btw)

Ive rebased it and it still seems to work, maybe time to merge this?
Btw it adds and activates jaspPredictiveAnalytics 

@petersen-f that is ok no?
@EJWagenmakers ?

**Comment by JorisGoosen** [comment]
Oh wait there is still a small problem with the order of top notifications

**Comment by JorisGoosen** [comment]
It should be fine now

**Comment by JorisGoosen** [comment]
@boutinb  do you think you can check this out?

File: Desktop/html/js/analyses.js [hunk 1]
```diff
@@ -164,14 +177,17 @@ JASPWidgets.Analyses = JASPWidgets.View.extend({
 			this.analyses.splice(toIndex, 0, this.analyses.splice(fromIndex, 1)[0])
 			this.views.splice(toIndex + 1, 0, this.views.splice(fromIndex + 1, 1)[0])
 
+			fromAnalysis.render()
+			toAnalysis.render()
+
```
Comment (boutinb) [inline Desktop/html/js/analyses.js:182]: What does have to do with this PR?

**Review by boutinb — CHANGES_REQUESTED** [review]
jaspBase is added in this commit... Should it be?

**Comment by JorisGoosen** [comment]
> jaspBase is added in this commit... Should it be?

Did you look at the commit?
Its the jaspReport branch as I mentioned: https://github.com/jasp-stats/jasp-desktop/pull/5064#issuecomment-1571929625

File: Desktop/html/js/analyses.js (hunk 1 above)
Comment (JorisGoosen) [reply Desktop/html/js/analyses.js:182]: this is to add topnotifications, when you click them you scroll to the analysis.
this is here to make the analsyses rerender themselves after they move so they can place their topnotification in the right z order

**Review by boutinb — APPROVED** [review]

**Pull request merged**